docs/getting-started-with-micro/what-is-micro/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "What is Snowplow Micro?"
+sidebar_position: 0
+description: "Snowplow Micro is a lightweight version of the Snowplow pipeline that’s great for getting familiar with Snowplow, debugging and testing."
+---
```

Comment:
Given there is no headers on this page, is there a way to make it full width? Looks a bit weird...

Reply:
Done